packages/app/vite.config.mjs

```diff
@@ -44,3 +45,22 @@ export default makeConfig({
     }),
   ],
 })
+
+// The module federation needs to be added to the plugins array first
```

Comment:
Why does it need to be added first?

Reply:
I don't know to be honest, but it seems to not play nicely with the other plugins when it comes last.

Reply:
```suggestion
// With some trial and error, it appears that the module federation plugin needs to be added 
// to the plugins array first so that the dynamic modules are available properly with respect 
// to the other plugins.
```